src/site/content/en/blog/media-session/index.md

```diff
@@ -0,0 +1,405 @@
+---
+title: Customizing media notifications and playback controls
```

Comment:
```suggestion
title: Customizing media notifications and playback controls with the Media Session API
```

"How to do X with Y" is a good format for titles, where "X" is a generic description of the task and "Y" is the name of the specific feature/API. This is good SEO because some users search with general phrases whereas other users know the exact API that they need